src/lab_orchestrator_lib_auth/auth.py

```diff
@@ -28,7 +28,7 @@ class LabInstanceTokenParams:
 
 
 def generate_auth_token(user_id: Identifier, lab_instance_token_params: LabInstanceTokenParams,
-                        secret_key: str, expires_in: int = 600) -> str:
+                        secret_key: str, expires_in: int = 600, expires_at: int = None) -> str:
```

Comment:
because `expires_at` has a default to `None` the correct type hint is `Optional[int]`

Reply:
fixed